meilisearch-release-changelogs/v0.28.0.md

```diff
@@ -0,0 +1,173 @@
+- The `matches` search parameter (used when calling the [`GET/POST /indexes/{uid}/search` route](https://docs.meilisearch.com/reference/api/search.html)) is renamed `showMatchesPosition`
+- The `_matchesInfo` field in the response body (received when calling the [`GET/POST /indexes/{uid}/search` route](https://docs.meilisearch.com/reference/api/search.html)) is renamed into `_matchesPosition`
+- `exhaustiveNbHits` and `exhaustiveFacetsCount` are removed because not used.
+
+Done by @irevoire in #2406.
+
+### Changes in `/indexes` endpoints
+
+- When getting the index list, the indexes are now paginated. It means:
+  - You can use the `offset` and `limit` parameters to browse the indexes.
+  - The indexes are returned in the `result` field in the response body.
+  - The `limit`, `offset`, and `total` are also now available in the response body.
+- The endpoint to update an index is now `PATCH /indexes/{indexUid}` instead of `PUT`
+- The already deprecated `name` field returned in the response body is totally removed.
+
+Done by @irevoire and @Kerollmops in #2423, #2452, and #2454.
+
+### Changes in `indexes/{uid}/documents` endpoints
+
+- When getting the documents via the `GET /indexes/{uid}/documents` route, the pagination has been improved. It means:
+  - You can still use the `offset` and `limit` parameters to browse the indexes.
+  - The documents are returned in the `result` field in the response body.
+  - The `limit`, `offset`, and `total` are also now available in the response body.
+- When getting the documents via the `GET /indexes/{uid}/documents` route, the `attributesToRetrieve` parameter has been renamed into `fields`
+- The `fields` parameter is available for the `GET /indexes/{uid}/documents/{document_uid}` route.
+- The [`displayedAttributes` settings](https://docs.meilisearch.com/reference/api/displayed_attributes.html) does not impact the `indexes/{uid}/documents` anymore, but ony the `/indexes/{uid}/search` endpoints.
+
+Done by @irevoire in #2372, #2433, and #2454.
+
+### Changes in `/tasks` and `/indexes/{uid}/tasks` endpoints
+
+- Add filtering on the `GET /tasks` endpoint:
+  - Add filtering capabilities on `type`, `status` and `indexUid` for `GET` `task` lists endpoints.
+  - It is possible to specify several values for a filter using the `,` character. e.g. `?status=enqueued,processing`
+  - Between two different filters, an `AND` operation is applied. e.g. `?status=enqueued&type=indexCreation` is equivalent to `status=enqueued AND type = indexCreation`
+- Add keyset pagination on the `GET /tasks` endpoints, allowing users to browse multiple sets of `task` items.
+  - You can use the `limit`, `from`, `next` query parameters to browse the tasks.
+  - The tasks are still returned in the `result` field in the response body.
```

Comment:
Not sure it's worth mentioning things that don't change.

Reply:
You might be right, let's decide the docs-team reviewer!